File: C/Listas/Lista 2/ex02.c
//Faça um programa que leia dois valores nas variáveis A e B respectivamente,
//troque o valor contido na variável A pelo valor em B, e o valor em B pelo valor em A,
//isto é, imprimiremos A e B com os valores trocados.
#include <stdio.h>

int main(void){
    int a, b, c;
    printf("Insira o valor de A: ");
    scanf("%d", &a);
    printf("Insira o valor de B: ");
    scanf("%d", &b);
    c=a;
    a=b;
    b=c;
    printf("a = %d\nb = %d", a, b);
}

File: Java/extra/Lista 02/src/ex02.java
/* Faça um programa que leia dois valores nas variáveis A e B respectivamente, 
troque o valor contido na variável A pelo valor em B, e o valor em B pelo valor em A, 
isto é, imprimiremos A e B com os valores trocados. */

import java.util.Scanner;

public class ex02 {
    static void troca(int a, int b) {
        int t;
        t = a;
        a = b;
        b = t;

        System.out.printf("\nA = %d\nB = %d", a, b);
    }

    public static void main(String[] args) {
        int n1, n2;
        System.out.print("Informe o valor de A: ");
        try (Scanner leitor = new Scanner(System.in)) {
            n1 = leitor.nextInt();
            System.out.print("Informe o valor de B: ");
            n2 = leitor.nextInt();
        }
        troca(n1, n2);
    }
}